scripts/util.py

```diff
@@ -1002,10 +1064,12 @@ def ensure_execute(request):
         break
 
 
-def batch_execute(requests, compute=compute, retry_cb=None, log_err=log.error):
+def batch_execute(requests, compute=None, retry_cb=None, log_err=log.error):
```

Comment:
We never supply `compute` here, please remove it.

Reply:
Done.